plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -196,25 +196,23 @@ class Bazel(
         )
 }
 
-private fun ModuleMetadata.vcsInfo(): VcsInfo {
-    val repo = repository?.firstOrNull().orEmpty()
-
-    // From looking at all current values of this field on BCR, it looks like only the special value "github:"
-    // exists. Otherwise, it's just the repo URL.
-    val url = if (repo.startsWith("github:")) {
-        val path = repo.substringAfter("github:")
+private fun String.expandRepositoryUrl(): String =
+    // From looking at all current values of this field on BCR, it looks like only the special value "github:" exists.
+    // Otherwise, it's just the repo URL.
+    if (startsWith("github:")) {
+        val path = substringAfter("github:")
```

Comment:
Unrelated, but mind adding a commit that simplifies these two lines into a single use of `withoutPrefix("github:")?.let { ... }`?